account-info/src/call_set_url.rs

```diff
@@ -0,0 +1,23 @@
+#![no_main]
+
```

Comment:
Is this module needed?

Reply:
Do you mean the file or the `#![no_main]` ?

Reply:
@deuszex the file itself seems to redundant. Or maybe I'm missing something.